address-space-controller/src/main/java/io/enmasse/controller/PodDistruptionBudgetController.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.enmasse.controller;
+
+import io.enmasse.address.model.AddressSpace;
+import io.enmasse.admin.model.v1.InfraConfig;
+import io.enmasse.admin.model.v1.StandardInfraConfig;
+import io.enmasse.config.AnnotationKeys;
+import io.fabric8.kubernetes.api.model.IntOrString;
+import io.fabric8.kubernetes.api.model.policy.PodDisruptionBudget;
+import io.fabric8.kubernetes.api.model.policy.PodDisruptionBudgetBuilder;
+import io.fabric8.kubernetes.client.KubernetesClientException;
+import io.fabric8.kubernetes.client.NamespacedKubernetesClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PodDistruptionBudgetController implements Controller {
+    private static final Logger log = LoggerFactory.getLogger(PodDistruptionBudgetController.class);
+
+    private final NamespacedKubernetesClient client;
+    private final String namespace;
+
+    public PodDistruptionBudgetController(NamespacedKubernetesClient client, String namespace) {
+        this.client = client;
+        this.namespace = namespace;
+    }
+
+    public AddressSpace reconcileActive(AddressSpace addressSpace) throws Exception {
+        InfraConfig infraConfig = InfraConfigs.parseCurrentInfraConfig(addressSpace);
+
+        if (infraConfig instanceof StandardInfraConfig) {
+            RouterSet routerSet = RouterSet.create(namespace, addressSpace, client);
+            reconcileRouterPodDisruptionBudget(addressSpace, routerSet, (StandardInfraConfig) infraConfig);
+            reconcileBrokerPodDisruptionBudget(addressSpace, (StandardInfraConfig) infraConfig);
```

Comment:
@lulf Could PDBs also be created for brokers in a BrokeredInfraConfig?

Reply:
No, it doesn't make sense there as you can never have more than 1 broker.

Reply:
Ok, perfect. That makes perfect sense. I wasn't as familiar with brokered address spaces and didn't realise it could only create one broker.